packager.go

```diff
@@ -218,10 +218,8 @@ func NewPackager(cert *x509.Certificate, key crypto.PrivateKey, packagerBase str
 	if !acceptablePackagerSchemes[baseURL.Scheme] {
 		return nil, errors.Errorf("PackagerBase %q must be over http or https.", packagerBase)
 	}
-	validityURL, err := url.Parse("https://cdn.ampproject.org/null-validity")
-	if err != nil {
-		return nil, errors.Wrap(err, "parsing null-validity URL")
-	}
+	validityURL := baseURL
+	validityURL.Path = ValidityMapURL
```

Comment:
baseURL may contain path components, e.g. `https://example.com/foobar/` -- setting Path will overwrite that. Please append instead, so it looks like `https://example.com/foobar/amppkg/validity`. I use ResolveReference in genCertUrl to do this, but there may be a better way (`path.Join` or something?).

Reply:
Done